src/reference/asciidoc/debezium.adoc

```diff
@@ -79,8 +93,8 @@ public class DebeziumJavaApplication {
     }
 
     @ServiceActivator(inputChannel = "debeziumInputChannel")
-    public void handler(String changeEventData) {
-        System.out.println(changeEventData);
+    public void handler(Message<?> changeEventData) {
+        System.out.println(new String((byte[]) changeEventData.getPayload()));
```

Comment:
And why did you change this back to such a style?
Why just plain `handler(String changeEventData)` is not enough anymore?
Or at least `byte[] changeEventData` if you think that there is no automatic conversion to string...